packages/beacon-node/src/chain/regen/regen.ts

```diff
@@ -238,17 +264,16 @@ export class StateRegenerator implements IStateRegeneratorInternal {
           // also with allowDiskReload flag, we "reload" it to the state cache too
           this.modules.blockStateCache.add(state);
         }
-
-        // this avoids keeping our node busy processing blocks
-        await nextEventLoop();
```

Comment:
Is removing this intentional?

Reply:
yes, similar to the purpose of `Promise.all()` above, we want to save I/O time here. This line was added when we didn't have worker thread